src/main/java/com/wavefront/AbstractTraceLoader.java

```diff
@@ -28,7 +30,9 @@
   protected static final Logger LOGGER = Logger.getLogger("traceloader");
   protected final GeneratorConfig generatorConfig = new GeneratorConfig();
   protected ApplicationConfig applicationConfig;
-
+  public static MemoryMXBean memoryMXBean = ManagementFactory.getMemoryMXBean();
+  public static double usedHeapMemory = (double) memoryMXBean.getHeapMemoryUsage().getUsed() / 1073741824;
```

Comment:
I guess that 1073741824 is the number of Bytes in GB. If you need some constant, please create a final variable, it will help you to avoid problems with typos.
Also, rename your variables to indicate that this is in GB (e.g. usedHeapMemoryGB).